requirements.txt

```diff
@@ -0,0 +1,8 @@
+python-dotenv==0.20.0
+loguru==0.6.0
+
+git+https://github.com/boudinfl/pke.git # Keyword Extraction Toolkit
+
+# gRPC deps
+grpcio-tools==1.46.3
+googleapis-common-protos==1.56.1
```

Comment:
Regarding what you said about how `pip freeze` dumps all the packages in your env, I believe if you use a virtual environment it dumps everything project-scoped nah? I may be wrong, I don't know much about Python.

Reply:
Also, thank you for adding the versions.

Reply:
You could, but everything should be installed before hand you call `pip freeze`, and we already use this requirements.txt to install everything before hand.

Reply:
I don't think we're talking about the same thing. Let's go over this next time.

Reply:
I mean, i guess you're talking about python venv, but even if we use it we need to specify which packages to install, here, I chose to do so with a requirements.txt.